data/testdata/all_types.golden.json

```diff
@@ -108,7 +110,9 @@
       [true, false, null, true, false],
       [0, 1568039445000, 1568039450000, 9007199254, 9223372036854],
       [0, 1568039445000, 1568039450000, 9007199254, 9223372036854],
-      [0, 1568039445000, null, 9007199254, 9223372036854]
+      [0, 1568039445000, null, 9007199254, 9223372036854],
+			[{"a":1},[1,2,3],{"b":2},[{"c":3},{"d":4}],{"e":{"f":5}}],
+			[{"a":1},[1,2,3],null,[{"c":3},{"d":4}],{"e":{"f":5}}]
```

Comment:
let's hope this never actually happens in real life, even though it's technically valid JSON :sweat_smile: